Support both explicit kubeconfigs and file paths (#12)
giantswarm/apptest: CHANGELOG.md

## [Unreleased]

### Added

- Support both explicit kubeconfigs and file paths.

> 
> ### Changed
> 
> - Reduce interval when waiting for apps now app-operator has a status webhook.

Reviewer: I think it isn't a breaking change :)

```suggestion
- Optimize apps wait interval as app-operator has a status webhook.
```

Author: True :) that is much nicer and I changed the heading to Fixed. PTAL
